packages/flex-plugin-scripts/src/scripts/__tests__/start.test.ts

```diff
@@ -32,12 +32,58 @@ describe('StartScript', () => {
       process.env.PORT = '2345';
       await startScripts.default();
 
-      expect(findPorts).toHaveBeenCalledTimes(1);
-      expect(findPorts).toHaveBeenCalledWith(port);
+      expect(findPort).toHaveBeenCalledTimes(1);
+      expect(findPort).toHaveBeenCalledWith(port);
       expect(getDefaultPort).toHaveBeenCalledTimes(1);
       expect(getDefaultPort).toHaveBeenCalledWith('2345');
       expect(_startDevServer).toHaveBeenCalledTimes(1);
-      expect(_startDevServer).toHaveBeenCalledWith(port);
+      expect(_startDevServer).toHaveBeenCalledWith(port, WebpackType.Complete);
+    });
+
+    it('should start static html page', async () => {
+      const port = 1234;
+      const findPort = jest
+        .spyOn(urlScripts, 'findPort')
+        .mockResolvedValue(port);
+      const getDefaultPort = jest
+        .spyOn(urlScripts, 'getDefaultPort')
+        .mockReturnValue(port);
+      const _startDevServer = jest
+        .spyOn(startScripts, '_startDevServer')
+        .mockReturnThis();
+
+      process.env.PORT = '2345';
+      await startScripts.default('flex');
+
+      expect(findPort).toHaveBeenCalledTimes(1);
```

Comment:
The `expect` tests are duplicated. Instead, within the `describe('default')` closure (i.e between line 19 and 20) add:

```js
const assertTest = (port: number, type: WebpackType) => {
      expect(findPort).toHaveBeenCalledTimes(1);
      expect(findPort).toHaveBeenCalledWith(port);
      expect(getDefaultPort).toHaveBeenCalledTimes(1);
      expect(getDefaultPort).toHaveBeenCalledWith(p.toString());
      expect(_startDevServer).toHaveBeenCalledTimes(1);
      expect(_startDevServer).toHaveBeenCalledWith(port, type);
}
```

Then update all of your tests to instead simply do:

```js
process.env.PORT = port.toString();
await startScripts.default('flex');

assertTest(port, WebpackType.Static);
```